man/systemd.link.xml

```diff
@@ -697,6 +697,29 @@
           <para>Takes a integer. Specifies the NIC transmit ring buffer size. When unset, the kernel's default will be used.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>RxFlowControl=</varname></term>
+        <listitem>
+          <para>Takes a boolean. When set, enables the receive flow control, also known as the ethernet
+          receive PAUSE message (generate and send ethernet PAUSE frames). When unset, the kernel's
+          default will be used.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>TxFlowControl=</varname></term>
+        <listitem>
+          <para>Takes a boolean. When set, enables the transmit flow control, also known as the ethernet
+          transmit PAUSE message (respond to received ethernet PAUSE frames). When unset, the kernel's
+          default will be used.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>AutoNegotiationFlowControl=</varname></term>
+        <listitem>
+          <para>Takes a boolean. When set, enables the auto negotiation flow control, also known as the
+          auto negotiation PAUSE message. When unset, the kernel's default will be used.</para>
```

Comment:
Please be a bit more verbose here. What does setting this to true mean, and how does this relate to RxFlowControl and TxFlowControl.

Reply:
@ssahani Do you have any suggestion?

Reply:
Takes a boolean. When set, enables the auto negotiation flow control. Auto negotiation enables the interface to exchange state advertisements with the connected peer so that the two devices can agree on the Ethernet PAUSE configuration....

Reply:
Thanks.